### ISSUE TITLE
Rewards twitter inline tip button UI change

### ISSUE THREAD

**Issue by codybanxs** [brave/brave-browser#13273]
## Description 
The inline tip button for Twitter needs to be updated to closer reflect the existing icons around it. 

## Assets
Inactive:
<img width="927" alt="Screen Shot 2020-12-21 at 11 33 55 AM" src="https://user-images.githubusercontent.com/61210652/102816877-94228880-4383-11eb-9719-daf1cb7b2247.png">

Active:
<img width="927" alt="Screen Shot 2020-12-21 at 11 47 11 AM" src="https://user-images.githubusercontent.com/61210652/102816905-a270a480-4383-11eb-9e0f-9bd6255e9974.png">

**Comment by jenn-rhim** [comment]
This tip button change applies to inline tipping for all social platforms that we released the tipping functionality without the hover state.  This hover state is specific to the Twitter.

**Comment by zenparsing** [comment]
A follow-up issue has been created for adding the hover background circle: #13280.

**Comment by zenparsing** [comment]
This will be released in GL 1.0.40.

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave	1.18.77 Chromium: 87.0.4280.101 (Official Build) (x86_64)
Revision	9407c80213cda69c2b7abcb4fa8e3f74488f4956-refs/branch-heads/4280@{#1807}
OS	macOS Version 10.15.7 (Build 19H15)
```

Used dev component version 1.0.40:
<img width="355" alt="Component" src="https://user-images.githubusercontent.com/28145373/103698939-23c55b00-4f70-11eb-9c2b-91f517b62c74.png">

<details>
<summary>Confirmed updated in-line tip icon on Twitter</summary>

Per https://github.com/brave/brave-browser/issues/13273#issuecomment-749627825, hover change to be completed in a follow up issue https://github.com/brave/brave-browser/issues/13280.

Confirmed updated icon is displayed while logged in and not logged in to twitter:

Not logged in | Logged in
----- | -----
<img width="661" alt="Twitter - not logged in" src="https://user-images.githubusercontent.com/28145373/103698947-29bb3c00-4f70-11eb-9d77-28a7ca77487d.png"> | <img width="768" alt="Twitter - logged in" src="https://user-images.githubusercontent.com/28145373/103698955-2cb62c80-4f70-11eb-8c07-06d31e24857b.png">

</details>

<details>
<summary>Confirmed updated in-line tip icon on GitHub</summary>

Confirmed updated in-line tip icon on various GH pages:

Issue - https://github.com/brave/brave-browser/issues/13374
<img width="1192" alt="Screen Shot 2021-01-05 at 4 00 54 PM" src="https://user-images.githubusercontent.com/28145373/103698981-38095800-4f70-11eb-954a-313da2df1759.png">

PR - https://github.com/brave/brave-core/pull/7358
<img width="1194" alt="Screen Shot 2021-01-05 at 4 03 14 PM" src="https://user-images.githubusercontent.com/28145373/103698999-3e97cf80-4f70-11eb-9bd6-77f9df76e09e.png">

Commits page - https://github.com/brave/brave-core/commits
<img width="1194" alt="Screen Shot 2021-01-05 at 4 01 24 PM" src="https://user-images.githubusercontent.com/28145373/103699003-40fa2980-4f70-11eb-9c7d-b0eab040b7a1.png">

Gist - https://gist.github.com/LaurenWags/b22c1b0f1f8bace284f9d9c5bc36d3b9
<img width="1192" alt="Screen Shot 2021-01-05 at 4 02 13 PM" src="https://user-images.githubusercontent.com/28145373/103699006-42c3ed00-4f70-11eb-92b2-d611c7c1e6e6.png">

</details>

<details>
<summary>Confirmed updated in-line tip icon on Reddit</summary>

Confirmed updated in-line tip icon on various Reddit pages:

Subreddit - https://www.reddit.com/r/BATProject/
<img width="1193" alt="Screen Shot 2021-01-05 at 4 04 36 PM" src="https://user-images.githubusercontent.com/28145373/103699020-46f00a80-4f70-11eb-8fb9-7ca0b5950705.png">

Thread - https://old.reddit.com/r/BATProject/comments/kpo2od/weekly_bat_discussion_thread/
<img width="1195" alt="Screen Shot 2021-01-05 at 4 05 17 PM" src="https://user-images.githubusercontent.com/28145373/103699023-49526480-4f70-11eb-88d4-a807778a8313.png">

User page - https://www.reddit.com/user/cryptojennie
<img width="1191" alt="Screen Shot 2021-01-05 at 4 05 47 PM" src="https://user-images.githubusercontent.com/28145373/103699027-4bb4be80-4f70-11eb-9c44-9ffad5a9a7fa.png">

Main page - https://old.reddit.com/
<img width="1194" alt="Screen Shot 2021-01-05 at 4 09 28 PM" src="https://user-images.githubusercontent.com/28145373/103699091-6d15aa80-4f70-11eb-96f5-390b654d18ef.png">

</details>

Also performed "upgrade" case:
1. Launched 1.18.77 without `--use-dev-goupdater-url` flag to get component version 1.0.39
2. Visited various Twitter, Reddit, GitHub pages and confirmed current (colorful) in-line tip icon is displayed.
3. Closed Brave
4. Relaunched, using `--use-dev-goupdater-url` flag to get updated component version 1.0.40
5. Revisited same pages and confirmed updated (single color) in-line tip icon is now displayed.
